loader/loader.py

```diff
@@ -3,56 +3,53 @@
 "http://ghtorrent-downloads.ewi.tudelft.nl/mongo-daily/" and store them into target directory.
 """
 import argparse
+from datetime import datetime
+from functools import partial
+from multiprocessing import Pool
 import os
+import re
 import tarfile
-from typing import List, Callable, Optional
+from typing import List
 
 from bs4 import BeautifulSoup
+import requests
 from tqdm import tqdm
-import urllib.request
 
 
-def download_progress_hook(p_bar: tqdm) -> Callable[[int, int, Optional[int]], None]:
-    """
-    Wraps tqdm instance
-    :param p_bar: tqdm instance responible for showing download progress
-    :return: function update_to which updates tqdm state
-    """
-    last_block = [0]
-
-    def update_to(block_num: int = 1, block_size: int = 1,
-                  total_size: Optional[int] = None) -> None:
-        """
-        Update tqdm state
-        :param block_num: Number of blocks transferred so far
-        :param block_size: Size of each block
-        :param total_size: Total size (in tqdm units)
-        :return: None
-        """
-        if total_size not in (None, -1):
-            p_bar.total = total_size
-        p_bar.update((block_num - last_block[0]) * block_size)
-        last_block[0] = block_num
-
-    return update_to
-
-
-def extract_archive_links(url: str) -> List[str]:
+def extract_archive_links(url: str, start_date: str, end_date: str) -> List[str]:
     """
     Extract tar files links from GHTorrent html page to list
     :param url: url of page with daily dumps tars
+    :param start_date: Start date in ISO format
```

Comment:
for style consistancy - you should either start description with lower or upper case everywhere)